docs/developer/tutorials/snmp/profile-format.md

```diff
@@ -407,7 +407,7 @@ SNMP types not listed in this table are submitted as `gauge` by default.
 
 Sometimes the inferred type may not be what you want. Typically, OIDs that represent "total number of X" are defined as `Counter32` in MIBs, but you probably want to submit them `monotonic_count` instead of a `rate`.
 
-For such cases, you can define a `forced_type`. Possible values and their effect are listed below.
+For such cases, you can define a `metric_type`. Possible values and their effect are listed below.
 
 | Forced type                | Description                                                  |
```

Comment:
Should the heading on the table, as well as the section heading on line `406`, be updated as well?

Reply:
I think title "Forced metric types" should be fine since the features is still about "forcing" the "metric type".